library/coretests/tests/cell.rs

```diff
@@ -487,6 +487,32 @@ fn const_refcell() {
             panic!("should never be called");
         }
     }
+
+    // Check that `borrow` is usable at compile-time
+    const BORROW_TEST: RefCell<u32> = {
+        let a = RefCell::new(0);
+        {
+            assert!(a.try_borrow().is_ok());
+            assert!(a.try_borrow_mut().is_ok());
+            let _a = a.borrow();
+            assert!(a.try_borrow().is_ok());
+            assert!(a.try_borrow_mut().is_err());
+        }
+        a
+    };
+    // Check that `borrow` is usable at compile-time
```

Comment:
```suggestion
    // Check that `borrow_mut` is usable at compile-time
```
Also, please ensure that it is actually usable by doing reads and writes through the handle.